backend/src/build-system/__tests__/test.sms-lvl2.spec.ts

```
@@ -1,84 +1,55 @@
+import { isIntegrationTest } from 'src/common/utils';
+import { PRDHandler } from '../handlers/product-manager/product-requirements-document/prd';
+import { ProjectInitHandler } from '../handlers/project-init';
+import { UXDMDHandler } from '../handlers/ux/datamap';
+import { UXSMDHandler } from '../handlers/ux/sitemap-document';
+import { UXSMSHandler as UXSMSHandler } from '../handlers/ux/sitemap-structure';
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)